src/Exceptions/PusherException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace Pusher\Exceptions;
```

Comment:
The question is if we should remove it entirely or move it to the root. I think we should remove it all together.